Driverless AI/get-started-with-open-source-custom-recipes-tutorial/get-started-with-open-source-custom-recipes-tutorial.md

```diff
@@ -464,13 +466,30 @@ Confusion Matrices for each of the selected models
 
 ![project-model-comparison-matrix](assets/project-model-comparison-matrix.jpg)
 
-From the confusion matrices shown above, we can get an idea of how the models with the custom recipes improved. We can see that both, the Transformer and the Scorer recipes, yielded a lower misclassification error. However, since we were working with a slightly imbalanced dataset, we need to take a closer look at the confusion matrices. We see that by using the custom recipes, the models started predicting more **False** labels correctly, in other words, we see more **True Negatives;** thus, we have a smaller misclassification error for the **False** class. On the other hand, only the Scorer recipe helped our model to predict more **True** labels correctly because, for the Transformer recipe, it was the opposite. The Scorer recipe helped the model to get more **True Positives** than the baseline model, while the Transformer recipe got less **True Positives** than the baseline model.  For that reason, we see that the misclassification error for the True label only improved for the model with the Scorer recipe.
+From the confusion matrices shown above, we can get an idea of how the models with the custom recipes improved. We can see that the Scorer recipe yielded a lower misclassification error compared to the baseline misclassification error of 7.56% .
+
+- Formula: Misclassification (all **incorrect** / all) = FP + FN / TP + TN + FP + FN
+
+- The misclassification percentage for the Scorer recipe is:
+
+   Misclassification = 133 + 103 / 2717 + 380 + 133 + 103
```

Comment:
Can you please add parentheses to the following:
(133 + 103) / (2717 + 380 + 133 + 103)